src/utils/index.js

```diff
@@ -1,12 +1,13 @@
-// This file contains utility functions that are exposed game-side
+/** This file contains utility functions that are exposed game-side */
 
 /**
   * Choose a random number between a and b.
   * @example randRange([a, b], bipolar)
   * @example randRange(a, b, bipolar)
-  * @arg {number} a
-  * @arg {number} b
-  * @arg {boolean} [bipolar] - If true, then the returned value will be inverted half of the time
+  * @global
+  * @arg     {number}  a
+  * @arg     {number}  b
+  * @arg     {boolean} [bipolar=false] Should the returned value will be inverted half of the time
   * @returns {number}
   */
 randRange = function(a, b, bipolar) {
```

Comment:
@rogueSkib From [w3schools random](http://www.w3schools.com/jsref/jsref_random.asp) "Return a random number between 0 (inclusive) and 1 (exclusive)"

Some quick findings:
- rollInt does not follow this behavior: `rollInt(0, 1)` returns [0, 1]
- rollInt does not behave well with invalid min max: `rollInt(1, 0)` returns [1]
- rollFloat does not behave well with invalid min max: `rollFloat(1, 0)` returns 0 < x <= 1
`for (var i = 0; i < 10000000000; i++) { if (rollFloat(1, 0) === 1)  { console.log('bad') } }`